<ui version="4.0" >
 <class>ConsoleWidgetClass</class>
 <widget class="QWidget" name="ConsoleWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>XML Console</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <property name="margin" >
    <number>5</number>
   </property>
   <item>
    <widget class="QSplitter" name="sptHSplitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="grbFilter" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Filter</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="lblStreamJid" >
         <property name="text" >
          <string>Stream:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbStreamJid" />
       </item>
       <item>
        <widget class="Line" name="line_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCondition" >
         <property name="text" >
          <string>Condition:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbCondition" >
         <property name="editable" >
          <bool>true</bool>
         </property>
         <property name="insertPolicy" >
          <enum>QComboBox::InsertAlphabetically</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ltwConditions" />
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QToolButton" name="tlbAddCondition" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tlbRemoveCondition" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tlbClearCondition" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblContext" >
         <property name="text" >
          <string>Available context:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbContext" >
         <property name="editable" >
          <bool>true</bool>
         </property>
         <property name="insertPolicy" >
          <enum>QComboBox::InsertAlphabetically</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QToolButton" name="tlbLoadContext" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tlbSaveContext" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tlbDeleteContext" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="grbConsole" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Console</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <property name="margin" >
        <number>5</number>
       </property>
       <item>
        <widget class="QSplitter" name="sptVSplitter" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTextEdit" name="tedConsole" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>5</verstretch>
           </sizepolicy>
          </property>
          <property name="lineWrapMode" >
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="tedSendXML" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="undoRedoEnabled" >
           <bool>false</bool>
          </property>
          <property name="acceptRichText" >
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <widget class="QCheckBox" name="chbWordWrap" >
           <property name="text" >
            <string>Word wrap</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="tlbSendXML" >
           <property name="text" >
            <string>Send</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tlbClearConsole" >
           <property name="text" >
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
